package.json

```diff
@@ -104,6 +97,7 @@
     "rimraf": "3.0.2",
     "terminal-link": "2.1.1",
     "typescript": "4.7.4",
+    "typescript-transform-paths": "3.4.4",
```

Comment:
We don't need this one anymore; I'll get rid of it.

Reply:
https://github.com/redwoodjs/redwood/pull/4992/commits/900723965f3234af30151b259dd0b4ab43fa65bc